Dockerfile

```diff
@@ -105,7 +106,8 @@ RUN chmod +x /usr/local/bin/hamster && \
     chmod +x /usr/local/bin/delete-orphaned-projects && \
     chmod +x /usr/local/bin/clear-card-cache && \
     chmod +x /usr/local/bin/calc-users-stats && \
-    chmod +x /usr/local/bin/calc-tier-stats
+    chmod +x /usr/local/bin/calc-tier-stats && \
```

Comment:
Naming conventions has no clear pattern.

Reply:
We will be moving all the unrelated tasks to cloud repo where we will refactor the names